manifests/r/RaptorTech/RaptorHardwareService/2.3.6.0/RaptorTech.RaptorHardwareService.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: RaptorTech.RaptorHardwareService
+PackageVersion: 2.3.6.0
+InstallerType: burn
+Dependencies:
+  WindowsFeatures:
+    - netfx3 
+Installers:
+- InstallerUrl: https://cdn.raptortech.com/installers/stable/2_3_6_0/Raptor.HardwareServices_2_3_6_0.exe
+  Architecture: x86

```

```suggestion
  Architecture: x64
```

Our `x86` validations are performed on 32-bit OSes. From the errors, I'm guessing that either the package or its dependency are 64-bit only.